### PR TITLE
Added missing whitespace after numbered emojis in all languages [Fixe…

### LINKED ISSUE #12173: Bug report: missing whitespace
### Describe the bug

Missing space between emoji and text on design page

### To reproduce

1. Go to https://ethereum.org/en/contributing/design
2. Scroll down and notice list using emoji's for number
3. Notice lack of space between emoji and heading text

### Expected behavior

We should have some spacing between the emoji and the text. Commonly we'll use 0.5rem (chakra token `"2"`) for marginEnd (Chakra property `me`).

### Screenshots

image

### Desktop (please complete the following information)

_No response_

### Smartphone (please complete the following information)

_No response_

### Additional context

Inside `public/content/contributing/design/index.md` we should add `me="2"` to each Emoji (a margin _end_ of half a rem; do not use marginRight to ensure responsiveness for RTL languages)

This should be done for this page in all languages (`/public/content/translations/{lang}/community/online/index.md`)

### Would you like to work on this issue?

- [ ] Yes
- [X] No

### PR DESCRIPTION
**Pull request by RabihJabr29**
…s #12173]

Added missing whitespace after numbered emojis in all languages [Fixes #12173 ]

## Description

Issue #12173

Missing space between emoji and text on design page

Added the missing whitespace by having the property me="2" in the "Emoji" elements.
Done for all languages.

### CODE REVIEW

File: public/content/contributing/design/index.md [hunk 1]
```diff
@@ -12,31 +12,31 @@ Ultimately, this is a great way to build a diverse and impressive portfolio that
 
 ## How to contribute?
 
-### <Emoji text=":one:" size={1} /> Provide feedback on early design prototypes {#design-critique}
+### <Emoji text=":one:" size={1} me="2" /> Provide feedback on early design prototypes {#design-critique}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:15]:
```suggestion
### <Emoji text=":one:" size={1} /> &nbsp;Provide feedback on early design prototypes {#design-critique}
```

File: public/content/contributing/design/index.md [hunk 2]
```diff
@@ -12,31 +12,31 @@ Ultimately, this is a great way to build a diverse and impressive portfolio that
...
 
 We sometimes need a help testing our raw ideas. This is a great way how to contribute without any technical knowledge.
 
 1. The design team will share a mockup design on [Discord](https://discord.com/invite/ethereum-org) and on [GitHub](https://github.com/ethereum/ethereum-org-website/labels/design%20required%20%F0%9F%8E%A8).
 2. You will be guided through the designs to provide feedback via comments function.
 3. The outcome will be shared in the GitHub issue and then closed by the team.
 
-### <Emoji text=":two:" size={1} /> Participate in survey research {#answer-surveys}
+### <Emoji text=":two:" size={1} me="2" /> Participate in survey research {#answer-surveys}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:23]:
```suggestion
### <Emoji text=":two:" size={1} /> &nbsp;Participate in survey research {#answer-surveys}
```

File: public/content/contributing/design/index.md [hunk 3]
```diff
@@ -12,31 +12,31 @@ Ultimately, this is a great way to build a diverse and impressive portfolio that
...
 
 Provide feedback on our website by:
 
 1. Visiting ethereum.org and reading through pages.
 2. Clicking on the feedback widget at the bottom right corner and answering design and content-related questions.
 3. Focus on the free format questions.
 
-### <Emoji text=":three:" size={1} /> Find design related issues on the website and report them {#report-design-issues}
+### <Emoji text=":three:" size={1} me="2" /> Find design related issues on the website and report them {#report-design-issues}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:31]:
```suggestion
### <Emoji text=":three:" size={1} /> &nbsp;Find design related issues on the website and report them {#report-design-issues}
```

File: public/content/contributing/design/index.md [hunk 4]
```diff
@@ -12,31 +12,31 @@ Ultimately, this is a great way to build a diverse and impressive portfolio that
...
 
 Ethereum.org is a fast growing website with many features and content. Some of the UI can easily become obsolete or could be improved. If you encounter any such instance, please report it so that it gets our attention.
 
 1. Go through the website and pay attention to its design.
 2. Take screenshots and notes if you see any visual or UX issues.
 3. Report the found issues using a [bug report](https://github.com/ethereum/ethereum-org-website/issues/new/choose).
 
-### <Emoji text=":four:" size={1} /> Propose design changes {#propose-design-changes}
+### <Emoji text=":four:" size={1} me="2" /> Propose design changes {#propose-design-changes}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:39]:
```suggestion
### <Emoji text=":four:" size={1} /> &nbsp;Propose design changes {#propose-design-changes}
```

File: public/content/contributing/design/index.md [hunk 5]
```diff
@@ -45,7 +45,7 @@ If you feel comfortable taking on design challenges, you can visit our GitHub is
 3. Propose the solution in the corresponding GitHub issue or [create a new one.](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A&template=feature_request.yaml&title=Feature+request)
 4. Wait for the design team to review.
 
-### <Emoji text=":five:" size={1} /> Build Design System together {#Contribute-to-design-system}
+### <Emoji text=":five:" size={1} me="2" /> Build Design System together {#Contribute-to-design-system}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:48]:
```suggestion
### <Emoji text=":five:" size={1} /> &nbsp;Build Design System together {#Contribute-to-design-system}
```

File: public/content/contributing/design/index.md [hunk 6]
```diff
@@ -56,7 +56,7 @@ Our design system makes designing ethereum.org fun and easy. If you are an exper
 5. The design team will review.
 6. The Design team will incorporate the changes in the main file and publish the file to the community.
 
-### <Emoji text=":six:" size={1} /> Write design-related content on the website {#write-design-articles}
+### <Emoji text=":six:" size={1} me="2" /> Write design-related content on the website {#write-design-articles}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:59]:
```suggestion
### <Emoji text=":six:" size={1} /> &nbsp;Write design-related content on the website {#write-design-articles}
```

File: public/content/contributing/design/index.md [hunk 7]
```diff
@@ -66,7 +66,7 @@ The Ethereum developer community is strong, but the design community is falling
 4. Once approved, write the content.
 5. Submit it in the corresponding GH issue.
 
-### <Emoji text=":seven:" size={1} /> Draw new illustrations {#prepare-illustrations}
+### <Emoji text=":seven:" size={1} me="2" /> Draw new illustrations {#prepare-illustrations}
```
Comment (corwintines) [inline public/content/contributing/design/index.md:69]:
```suggestion
### <Emoji text=":seven:" size={1} /> &nbsp;Draw new illustrations {#prepare-illustrations}
```

File: public/content/translations/de/contributing/design/index.md [hunk 8]
```diff
@@ -12,31 +12,31 @@ Letztendlich ist das eine großartige Möglichkeit, ein vielfältiges und beeind
 
 ## Wie kann ich etwas beitragen?
 
-### <Emoji text=":one:" size={1} /> Geben Sie Feedback zu frühen Design-Prototypen {#design-critique}
+### <Emoji text=":one:" size={1} me="2" /> Geben Sie Feedback zu frühen Design-Prototypen {#design-critique}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:15]:
```suggestion
### <Emoji text=":one:" size={1} /> &nbsp;Geben Sie Feedback zu frühen Design-Prototypen {#design-critique}
```

File: public/content/translations/de/contributing/design/index.md [hunk 9]
```diff
@@ -12,31 +12,31 @@ Letztendlich ist das eine großartige Möglichkeit, ein vielfältiges und beeind
...
 
 Manchmal brauchen wir Hilfe beim Testen unserer "rohen" Ideen. Das ist eine großartige Möglichkeit, auch ohne technische Kenntnisse einen Beitrag zu leisten.
 
 1. Das Design-Team wird ein Mockup-Design auf [Discord](https://discord.com/invite/CetY6Y4) und auf [GitHub](https://github.com/ethereum/ethereum-org-website/labels/design%20required%20%F0%9F%8E%A8) veröffentlichen.
 2. Sie werden durch die Entwürfe geführt und können über die Kommentarfunktion Feedback geben.
 3. Das Ergebnis wird in einem GitHub-Issue geteilt und dann vom Team abgeschlossen.
 
-### <Emoji text=":two:" size={1} /> Teilnahme an Umfragen {#answer-surveys}
+### <Emoji text=":two:" size={1} me="2" /> Teilnahme an Umfragen {#answer-surveys}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:23]:
```suggestion
### <Emoji text=":two:" size={1} /> &nbsp;Teilnahme an Umfragen {#answer-surveys}
```

File: public/content/translations/de/contributing/design/index.md [hunk 10]
```diff
@@ -12,31 +12,31 @@ Letztendlich ist das eine großartige Möglichkeit, ein vielfältiges und beeind
...
 
 Geben Sie Feedback zu unserer Website:
 
 1. Besuchen Sie ethereum.org und lesen Sie die Seiten durch.
 2. Klicken Sie auf das Feedback-Widget in der rechten unteren Ecke und beantworten Sie Fragen zum Design und zum Inhalt.
 3. Konzentrieren Sie sich auf die Fragen zum freien Format.
 
-### <Emoji text=":three:" size={1} /> Finden Sie designbezogene Probleme auf der Website und melden Sie diese. {#report-design-issues}
+### <Emoji text=":three:" size={1} me="2" /> Finden Sie designbezogene Probleme auf der Website und melden Sie diese. {#report-design-issues}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:31]:
```suggestion
### <Emoji text=":three:" size={1} /> &nbsp;Finden Sie designbezogene Probleme auf der Website und melden Sie diese. {#report-design-issues}
```

File: public/content/translations/de/contributing/design/index.md [hunk 11]
```diff
@@ -12,31 +12,31 @@ Letztendlich ist das eine großartige Möglichkeit, ein vielfältiges und beeind
...
 
 Ethereum.org ist eine schnell wachsende Website mit vielen Funktionen und Inhalten. Einige der Benutzeroberflächen können leicht veraltet sein oder verbessert werden. Wenn Ihnen ein solches Problem auffällt, melden Sie es bitte, damit wir darauf aufmerksam werden.
 
 1. Gehen Sie durch die Website und achten Sie auf ihr Design.
 2. Machen Sie Screenshots und Notizen, wenn Sie visuelle oder UX-Probleme sehen.
 3. Melden Sie die gefundenen Probleme in einem [Fehlerbericht](https://github.com/ethereum/ethereum-org-website/issues/new/choose).
 
-### <Emoji text=":four:" size={1} /> Designänderungen vorschlagen {#propose-design-changes}
+### <Emoji text=":four:" size={1} me="2" /> Designänderungen vorschlagen {#propose-design-changes}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:39]:
```suggestion
### <Emoji text=":four:" size={1} /> &nbsp;Designänderungen vorschlagen {#propose-design-changes}
```

File: public/content/translations/de/contributing/design/index.md [hunk 12]
```diff
@@ -45,7 +45,7 @@ Wenn Sie sich mit Design-Herausforderungen wohlfühlen, können Sie unser GitHub
 3. Schlagen Sie die Lösung in dem entsprechenden GitHub-Thema vor oder erstellen Sie ein [neues Thema](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A&template=feature_request.yaml&title=Feature+request).
 4. Warten Sie auf die Überprüfung durch das Designteam.
 
-### <Emoji text=":five:" size={1} /> Das Designsystem gemeinsam aufbauen {#Contribute-to-design-system}
+### <Emoji text=":five:" size={1} me="2" /> Das Designsystem gemeinsam aufbauen {#Contribute-to-design-system}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:48]:
```suggestion
### <Emoji text=":five:" size={1} /> &nbsp;Das Designsystem gemeinsam aufbauen {#Contribute-to-design-system}
```

File: public/content/translations/de/contributing/design/index.md [hunk 13]
```diff
@@ -56,7 +56,7 @@ Mit unserem Designsystem macht das Entwerfen von ethereum.org Spaß und ist einf
 5. Das Designteam wird es dann überprüfen.
 6. Das Designteam wird die Änderungen in die Hauptdatei einarbeiten und die Datei in der Community veröffentlichen.
 
-### <Emoji text=":six:" size={1} /> Verfassen Sie designbezogene Inhalte auf der Website. {#write-design-articles}
+### <Emoji text=":six:" size={1} me="2" /> Verfassen Sie designbezogene Inhalte auf der Website. {#write-design-articles}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:59]:
```suggestion
### <Emoji text=":six:" size={1} /> &nbsp;Verfassen Sie designbezogene Inhalte auf der Website. {#write-design-articles}
```

File: public/content/translations/de/contributing/design/index.md [hunk 14]
```diff
@@ -66,7 +66,7 @@ Die Ethereum-Entwickler-Community ist stark, aber die Design-Community hinkt etw
 4. Sobald die Anfrage genehmigt ist, schreiben Sie den Inhalt.
 5. Reichen Sie ihn im entsprechenden GH-Thema ein.
 
-### <Emoji text=":seven:" size={1} /> Gestalten Sie neue Illustrationen. {#prepare-illustrations}
+### <Emoji text=":seven:" size={1} me="2" /> Gestalten Sie neue Illustrationen. {#prepare-illustrations}
```
Comment (corwintines) [inline public/content/translations/de/contributing/design/index.md:69]:
```suggestion
### <Emoji text=":seven:" size={1} /> &nbsp;Gestalten Sie neue Illustrationen. {#prepare-illustrations}
```

File: public/content/translations/es/contributing/design/index.md [hunk 15]
```diff
@@ -12,31 +12,31 @@ Por último, esto es un buen camino para construir un diverso e impresionante po
 
 ## ¿Cómo contribuir?
 
-### <Emoji text=":one:" size={1} /> Proporciona opiniones en diseños tempranos de prototipos {#design-critique}
+### <Emoji text=":one:" size={1} me="2" /> Proporciona opiniones en diseños tempranos de prototipos {#design-critique}
```
Comment (corwintines) [inline public/content/translations/es/contributing/design/index.md:15]:
```suggestion
### <Emoji text=":one:" size={1} /> &nbsp;Proporciona opiniones en diseños tempranos de prototipos {#design-critique}
```

File: public/content/translations/es/contributing/design/index.md [hunk 16]
```diff
@@ -12,31 +12,31 @@ Por último, esto es un buen camino para construir un diverso e impresionante po
...
 
 Algunas veces necesitamos ayuda para testear nuestras ideas crudas. Esto es un buen camino para saber cómo contribuir sin algunos conocimientos técnicos.
 
 1. El equipo de diseño compartirá una maqueta de diseño en [Discord](https://discord.com/invite/CetY6Y4) y en [GitHub](https://github.com/ethereum/ethereum-org-website/labels/design%20required%20%F0%9F%8E%A8).
 2. Usted será guiado a través de diseños para proporcionar opiniones por comentarios de función.
 3. El resultado será compartido en el asunto GitHub y después será cerrado por el equipo.
 
-### <Emoji text=":two:" size={1} /> Participa en las encuentas {#answer-surveys}
+### <Emoji text=":two:" size={1} me="2" /> Participa en las encuentas {#answer-surveys}
```
Comment (corwintines) [inline public/content/translations/es/contributing/design/index.md:23]:
```suggestion
### <Emoji text=":two:" size={1} /> &nbsp;Participa en las encuentas {#answer-surveys}
```

File: public/content/translations/es/contributing/design/index.md [hunk 17]
```diff
@@ -12,31 +12,31 @@ Por último, esto es un buen camino para construir un diverso e impresionante po
...
 
 Proporciona opiniones en nuestra página web por:
 
 1. Vistando ethereum.org y leyendo a través de páginas.
 2. Clicleando en la ventana de opiniones en abajo a la derecha en la esquina y respondiedo diseños y preguntas de contenido relacionado.
 3. Céntrate en el formato de preguntas libres.
 
-### <Emoji text=":three:" size={1} /> Si detecta errores de diseño en la página web, comuníquelos {#report-design-issues}
+### <Emoji text=":three:" size={1} me="2" /> Si detecta errores de diseño en la página web, comuníquelos {#report-design-issues}
```
Comment (corwintines) [inline public/content/translations/es/contributing/design/index.md:31]:
```suggestion
### <Emoji text=":three:" size={1} /> &nbsp;Si detecta errores de diseño en la página web, comuníquelos {#report-design-issues}
```

File: public/content/translations/pt-br/contributing/design/index.md [hunk 18]
```diff
@@ -66,7 +66,7 @@ A comunidade de desenvolvedores Ethereum é forte, mas a comunidade de design es
 4. Uma vez aprovado, escreva o conteúdo.
 5. Envie-o no problema GH correspondente.
 
-### <Emoji text=":seven:" size={1} /> Desenhe novas ilustrações {#prepare-illustrations}
+### <Emoji text=":seven:" size={1} me="2" /> Desenhe novas ilustrações {#prepare-illustrations}
```
Comment (corwintines) [inline public/content/translations/pt-br/contributing/design/index.md:69]:
```suggestion
### <Emoji text=":seven:" size={1} /> &nbsp;Desenhe novas ilustrações {#prepare-illustrations}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 19]
```diff
@@ -12,31 +12,31 @@ Sonuç olarak bu, tasarım becerilerinizi sergileyeceğiniz çeşit çeşit ve e
 
 ## Nasıl katkıda bulunulur?
 
-### <Emoji text=":one:" size={1} /> Erken tasarım prototipleri hakkında geribildirim sağlayın {#design-critique}
+### <Emoji text=":one:" size={1} me="2" /> Erken tasarım prototipleri hakkında geribildirim sağlayın {#design-critique}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:15]:
```suggestion
### <Emoji text=":one:" size={1} /> &nbsp;Erken tasarım prototipleri hakkında geribildirim sağlayın {#design-critique}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 20]
```diff
@@ -12,31 +12,31 @@ Sonuç olarak bu, tasarım becerilerinizi sergileyeceğiniz çeşit çeşit ve e
...
 
 Bazen ham fikirlerimizi test etme konusunda yardıma ihtiyaç duyuyoruz. Bu, herhangi bir teknik bilgi olmadan katkıda bulunmanın harika bir yoludur.
 
 1. Tasarım ekibi, [Discord](https://discord.com/invite/CetY6Y4) ve [GitHub](https://github.com/ethereum/ethereum-org-website/labels/design%20required%20%F0%9F%8E%A8) üzerinden bir örnek tasarım paylaşacaktır.
 2. Tasarımların üzerinden yorumlar seçeneğiyle geribildirim vermeye yönlendirileceksiniz.
 3. Sonuç, Github konusunda paylaşılacak ve ekip tarafından kapatılacaktır.
 
-### <Emoji text=":two:" size={1} /> Anket araştırmasına katılın {#answer-surveys}
+### <Emoji text=":two:" size={1} me="2" /> Anket araştırmasına katılın {#answer-surveys}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:23]:
```suggestion
### <Emoji text=":two:" size={1} /> &nbsp;Anket araştırmasına katılın {#answer-surveys}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 21]
```diff
@@ -12,31 +12,31 @@ Sonuç olarak bu, tasarım becerilerinizi sergileyeceğiniz çeşit çeşit ve e
...
 
 Web sitemiz hakkında geribildirim sağlayın:
 
 1. Ethereum.org'u ziyaret edin ve sayfaları baştan sona okuyun.
 2. Sağ alt köşede bulunan geribildirim aracına tıklayıp tasarım ve içerik ile ilgili soruları cevaplayın.
 3. Serbest biçimdeki sorulara odaklanın.
 
-### <Emoji text=":three:" size={1} /> Web sitesindeki tasarımla ilgili sorunları bulun ve bildirin {#report-design-issues}
+### <Emoji text=":three:" size={1} me="2" /> Web sitesindeki tasarımla ilgili sorunları bulun ve bildirin {#report-design-issues}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:31]:
```suggestion
### <Emoji text=":three:" size={1} /> &nbsp;Web sitesindeki tasarımla ilgili sorunları bulun ve bildirin {#report-design-issues}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 22]
```diff
@@ -12,31 +12,31 @@ Sonuç olarak bu, tasarım becerilerinizi sergileyeceğiniz çeşit çeşit ve e
...
 
 Ethereum.org, birçok içeriğe ve özelliğe sahip, hızla büyüyen bir web sitesidir. Bazı kullanıcı arayüzleri, kolayca güncelliğini yitirebilir ya da geliştirilebilir. Böyle bir durumla karşılaşırsanız, dikkate almamız için lütfen bize bildirin.
 
 1. Web sitesine gidin ve tasarımları dikkatle inceleyin.
 2. Herhangi bir görsel sorun ya da kullanıcı deneyimi sorunu görürseniz, ekran görüntüsü ve notlar alın.
 3. Bulduğunuz sorunları [hata raporu](https://github.com/ethereum/ethereum-org-website/issues/new/choose) ile bildirin.
 
-### <Emoji text=":four:" size={1} /> Tasarım değişiklikleri önerin {#propose-design-changes}
+### <Emoji text=":four:" size={1} me="2" /> Tasarım değişiklikleri önerin {#propose-design-changes}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:39]:
```suggestion
### <Emoji text=":four:" size={1} /> &nbsp;Tasarım değişiklikleri önerin {#propose-design-changes}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 23]
```diff
@@ -45,7 +45,7 @@ Tasarım zorluklarının üstesinden gelme konusunda kendinizi rahat hissediyors
 3. İlgili GitHub konusunda çözümü önerin ya da [yeni bir tane oluşturun.](https://github.com/ethereum/ethereum-org-website/issues/new?assignees=&labels=feature+%3Asparkles%3A&template=feature_request.yaml&title=Feature+request)
 4. İncelemesi için tasarım ekibini bekleyin.
 
-### <Emoji text=":five:" size={1} /> Tasarım Sistemini birlikte geliştirin {#Contribute-to-design-system}
+### <Emoji text=":five:" size={1} me="2" /> Tasarım Sistemini birlikte geliştirin {#Contribute-to-design-system}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:48]:
```suggestion
### <Emoji text=":five:" size={1} /> &nbsp;Tasarım Sistemini birlikte geliştirin {#Contribute-to-design-system}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 24]
```diff
@@ -56,7 +56,7 @@ Tasarım sistemimiz, ethereum.org'u tasarlamayı eğlenceli ve kolay hale getiri
 5. Tasarım ekibi inceleyecektir.
 6. Tasarım ekibi, değişiklikleri ana dosyada toplayacak ve bu dosyayı topluluk ile paylaşacaktır.
 
-### <Emoji text=":six:" size={1} /> Web sitesinde tasarımla ilgili içerik yazın {#write-design-articles}
+### <Emoji text=":six:" size={1} me="2" /> Web sitesinde tasarımla ilgili içerik yazın {#write-design-articles}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:59]:
```suggestion
### <Emoji text=":six:" size={1} /> &nbsp;Web sitesinde tasarımla ilgili içerik yazın {#write-design-articles}
```

File: public/content/translations/tr/contributing/design/index.md [hunk 25]
```diff
@@ -66,7 +66,7 @@ Ethereum geliştirici topluluğu güçlü olsa da, tasarım topluluğu biraz dah
 4. Onaylandıktan sonra içeriği yazın.
 5. İlgili GH konusunda sunun.
 
-### <Emoji text=":seven:" size={1} /> Yeni illüstrasyonlar çizin {#prepare-illustrations}
+### <Emoji text=":seven:" size={1} me="2" /> Yeni illüstrasyonlar çizin {#prepare-illustrations}
```
Comment (corwintines) [inline public/content/translations/tr/contributing/design/index.md:69]:
```suggestion
### <Emoji text=":seven:" size={1} /> &nbsp;Yeni illüstrasyonlar çizin {#prepare-illustrations}
```

**Review by corwintines — CHANGES_REQUESTED** [review]
Thanks @RabihJabr29!

Ive left some suggestions which I think will be a cleaner solution. Instead of using margins, we can use the the character code for a space instead to get this. HTML is removing the white spaces between the Emoji and text, but if we add this character code it will be included.

**Comment by RabihJabr29** [comment]
Hello again, @corwintines 
Adapted the new changes to your suggestions.

**Review by corwintines — APPROVED** [review]
Thanks @RabihJabr29!

**Pull request merged**